src/jit/codegencommon.cpp

```diff
@@ -8734,6 +8774,22 @@ void CodeGen::genFnEpilog(BasicBlock* block)
                     }
                     break;
 
+                case IAT_RELPVALUE:
+#ifdef FEATURE_NGEN_RELOCS_OPTIMIZATIONS
```

Comment:
We do not use FEATURE ifdefs for cases like these when the VM asked the JIT to do something. We just leave the handling in unconditionally, even though it creates a bit of unreachable code in the JIT. It simplifies the matrix of different build combinations we need to worry about. It is nice to have one JIT library that can handle all of them (per host/target combination).